buildrunner/config.py

```diff
@@ -0,0 +1,405 @@
+"""
```

Comment:
theres quite a bit going on in this file, was this code moved ? I don't see a file. I'm mainly wondering if this is all new, should it have some more testing around it ?

Reply:
never mind, I found it.